src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,24 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val result: MutableList<String> = mutableListOf()
+        val greeting: HelloWorldGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world, Salut tout le monde, Hola, mundo"
```

Comment:
Please check task requirements there should be separate methods.
Also this method may be single-line.
And please check hardcoded strings